### ISSUE TITLE
[BUG] Clipping of axon bundles in axon map plotter

### ISSUE THREAD

**Issue by mbeyeler** [pulse2percept/pulse2percept#243]
The axon map [plot method](https://github.com/pulse2percept/pulse2percept/blob/master/pulse2percept/models/beyeler2019.py#L523) clips off axon segments outside the plotting range. This is fine, but in the case of a bundle leaving the plotting range and re-entering it later, the exit and re-entry point get connected with a straight line:

![image](https://user-images.githubusercontent.com/5214334/89442399-a0bf8080-d703-11ea-88c0-e76dd0a22600.png)

(Note the horizontal line at the bottom)

Instead, axon segments outside the plotting range should be set to NaN [here](https://github.com/pulse2percept/pulse2percept/blob/master/pulse2percept/models/beyeler2019.py#L573):

```
# Draw axon pathways:
for bundle in axon_bundles:
    # Set segments outside the drawing window to NaN:
    x_idx = np.logical_or(bundle[:, 0] < xmin, bundle[:, 0] > xmax)
    bundle[x_idx, 0] = np.nan
    y_idx = np.logical_or(bundle[:, 1] < ymin, bundle[:, 1] > ymax)
    bundle[y_idx, 1] = np.nan
    ax.plot(bundle[:, 0], bundle[:, 1], c=(0.6, 0.6, 0.6),
            linewidth=2, zorder=1)
```

**Comment by aiwenxu** [comment]
claiming this issue :)

**Comment by mbeyeler** [comment]
Closed via #244
